### PR TITLE
Make it possible to perform LxLy rotations

### PR DESCRIPTION
**Pull request by fretchen**
Proposal to implement the LxLy gate

### CODE REVIEW

File: qiskit_cold_atom/spins/spins_gate_library.py [hunk 1]
```diff
@@ -316,3 +316,31 @@ def generator(self) -> SpinOp:
 def lzz(self, gamma: float, wires: List[int], label=None):
     """Add the LZZ gate to a QuantumCircuit."""
     return self.append(LZZGate(gamma=gamma, label=label), qargs=wires)
+
+class LxLyGate(SpinGate):
+    r"""The spin exchange gate o two collective spins.
```
Comment (eggerdj) [inline qiskit_cold_atom/spins/spins_gate_library.py:321]:
```suggestion
    r"""The spin exchange gate of two collective spins.
```

**Review by eggerdj — CHANGES_REQUESTED** [review]
This looks good. Would it be possible to add a test, e.g. something like this? https://github.com/Qiskit-Extensions/qiskit-cold-atom/blob/b194142da8c9ed34566ae708b6f2c7350364fd7a/test/test_spin_gates.py#L35-L62

**Comment by fretchen** [comment]
Done. However, I was not try to find any indication anywhere on how to run the tests locally.

**Comment by LaurinFischer** [comment]
Hi Fred, this looks good, thanks. To run the tests, simply run 
```
python -m unittest 
```
from the repository root. (I added this to the Readme, thanks for pointing out.)

**Comment by fretchen** [comment]
So when I ran the test I found some issues with how I had implemented it and now it also uses `np.allclose()`. However, the tests fail as I did not add the Heidelberg provider credentials:

```
% python -m unittest
/qiskit-cold-atom/qiskit_cold_atom/providers/cold_atom_provider.py:289: UserWarning: No credentials found at the specified location
  warnings.warn("No credentials found at the specified location")
```

This is most likely an independent issue, but I it is somehow not obvious that this is good as I think that I mostly followed the most basic installation rules...

**Comment by LaurinFischer** [comment]
Hi Fred, I opened a new issue for the behaviour of the credentials in the test suite. I'm not sure what's going on as the test is designed such that it creates some dummy credentials on the fly and then deletes them afterwards. 
Do your test fail or is it just this warning which is being triggered? Maybe you could post the entire output of your console in the discussion of the issue #10

**Review by eggerdj — APPROVED** [review]
Looks good to me.

**Pull request merged**